SessionInstrumentation interface referenced in Getting Started not exported

## Description

If you try to implement the code snippet in the [Getting Started](https://github.com/grafana/faro-web-sdk/blob/main/docs/sources/tutorials/quick-start-browser.md#advanced) README, you will have an undefined reference to the `SessionInstrumentation` interface due to it not being exported in `packages/web-sdk/src/index.ts`.

## Steps to reproduce

1. Clone my [example repo](https://github.com/LinusBF/faro-web-sdk-issue-demo)
2. `yarn`
3. `yarn typecheck`

## Expected behavior

Implementing the example snippet should not result in any type or import errors

## Actual behavior

```
yarn run v1.22.19
$ tsc -p tsconfig.json --noEmit
index.ts:8:5 - error TS2724: '"@grafana/faro-web-sdk"' has no exported member named 'SessionInstrumentation'. Did you mean 'Instrumentation'?

8     SessionInstrumentation,
      ~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in index.ts:8

error Command failed with exit code 2.
```

## Environment

- **SDK version:** 1.0.2
- **SDK instrumentations:** Web SDK
- **Device type:** desktop
- **Device name:** Macbook Pro
- **OS:** MacOS
- **Browser:** N/A

--- a/packages/web-sdk/src/index.ts
+++ b/packages/web-sdk/src/index.ts
@@ -14,6 +14,7 @@ export {
   parseStacktrace,
   ViewInstrumentation,
   WebVitalsInstrumentation,
+  SessionInstrumentation,
 } from './instrumentations';
 export type { ConsoleInstrumentationOptions, ErrorEvent, ExtendedPromiseRejectionEvent } from './instrumentations';
 

